MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -643,11 +614,19 @@ private void ParseDlnaHeaders(StreamRequest request)
                 return null;
             }
 
-            if (value.IndexOf("npt=", StringComparison.OrdinalIgnoreCase) != 0)
+            if (!value.StartsWith("npt=", StringComparison.OrdinalIgnoreCase))
             {
                 throw new ArgumentException("Invalid timeseek header");
             }
-            value = value.Substring(4).Split(new[] { '-' }, 2)[0];
+            int index = value.IndexOf('-');
+            if (index == -1)
+            {
+                value = value.Substring(4);
+            }
+            else
+            {
+                value = value.Substring(4, index);
```

Comment:
huh, what this does? I'm not sure it is equivalent to the older code.
Also what is the reason for the change?

Reply:
This strips the "npt=" from the beginning of the string and everything after '-'.
I changed it because `Split` is kind of hacky for this (at least imo)

Reply:
Add a comment explaining that `4 == len("npt=")` please :)

Reply:
Made it self-documenting ;)